static/src/javascripts/projects/common/modules/lazyload.js

```diff
@@ -43,9 +44,13 @@ define([
                         crossOrigin: true
                     }).then(
                         function (resp) {
-                            into.html(opts.beforeInsert(resp.html));
-                            into.addClass('lazyloaded');
-                            opts.success(resp);
+                            var componentHtml = opts.beforeInsert(resp.html);
+                            fastdom.write(function () {
+                                into.html(componentHtml);
+                                into.addClass('lazyloaded');
+                                opts.success(resp);
```

Comment:
Again to PR person. Tests fail due to race conditions if this is outside `fastdom(...)` but I want as little as possible in there. is this just an imagined problem and should I get over it?